packages/react/src/Button/__tests__/__snapshots__/Button.test.tsx.snap

```diff
@@ -1680,6 +1680,7 @@ exports[`Button styles invisible button appropriately 1`] = `
   cursor: not-allowed;
   box-shadow: none;
   color: primer.fg.disabled;
+  background-color: var(--button-invisible-bgColor-disabled,undefined);
```

Comment:
What should the fallback value here be? Seems to be showing up as `undefined` 🤔

Reply:
Oh weird... lemme fix that